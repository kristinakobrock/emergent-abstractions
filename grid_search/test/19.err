  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 20.91it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:02, 19.75it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 23.32it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 23.80it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 29.44it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 26.65it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 25.11it/s] 52%|###############################################8                                            | 26/50 [00:01<00:00, 26.22it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 25.91it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 26.75it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 24.17it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 25.61it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 23.60it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 23.22it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 22.83it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.57it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 24.92it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 25.08it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 25.24it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 26.92it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.57it/s]
