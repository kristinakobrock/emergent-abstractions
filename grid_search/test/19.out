# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7081459760665894, "acc": 0.571134626865387, "length": 3.4908649921417236, "mode": "train", "epoch": 1}
{"loss": 0.6792675256729126, "acc": 0.5886719226837158, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6700237393379211, "acc": 0.6179946660995483, "length": 3.277513265609741, "mode": "train", "epoch": 2}
{"loss": 0.6775512099266052, "acc": 0.5810546875, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6641467809677124, "acc": 0.6132414937019348, "length": 2.7021942138671875, "mode": "train", "epoch": 3}
{"loss": 0.6663777232170105, "acc": 0.62109375, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6608985662460327, "acc": 0.6279458403587341, "length": 1.795422077178955, "mode": "train", "epoch": 4}
{"loss": 0.6665365695953369, "acc": 0.6259765625, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6584027409553528, "acc": 0.627821147441864, "length": 1.6704127788543701, "mode": "train", "epoch": 5}
{"loss": 0.660394549369812, "acc": 0.6259765028953552, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6566942930221558, "acc": 0.6270273923873901, "length": 1.8453649282455444, "mode": "train", "epoch": 6}
{"loss": 0.6596622467041016, "acc": 0.6259765028953552, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6551030278205872, "acc": 0.626660168170929, "length": 1.8555132150650024, "mode": "train", "epoch": 7}
{"loss": 0.6606791019439697, "acc": 0.6259765028953552, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6534420847892761, "acc": 0.6284217834472656, "length": 1.8517357110977173, "mode": "train", "epoch": 8}
{"loss": 0.6619460582733154, "acc": 0.6259765028953552, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6465939283370972, "acc": 0.6355475783348083, "length": 2.061673879623413, "mode": "train", "epoch": 9}
{"loss": 0.6472172737121582, "acc": 0.6488021016120911, "length": 1.41015625, "mode": "test", "epoch": 9}
{"loss": 0.6362948417663574, "acc": 0.6527665853500366, "length": 2.123110294342041, "mode": "train", "epoch": 10}
{"loss": 0.6295907497406006, "acc": 0.6583572030067444, "length": 1.97265625, "mode": "test", "epoch": 10}
{"loss": 0.6213404536247253, "acc": 0.6682129502296448, "length": 2.2745509147644043, "mode": "train", "epoch": 11}
{"loss": 0.6168655157089233, "acc": 0.66901695728302, "length": 2.1171875, "mode": "test", "epoch": 11}
{"loss": 0.6087464690208435, "acc": 0.67997807264328, "length": 2.2127256393432617, "mode": "train", "epoch": 12}
{"loss": 0.6021958589553833, "acc": 0.6793366074562073, "length": 2.484375, "mode": "test", "epoch": 12}
{"loss": 0.5964022278785706, "acc": 0.6832032203674316, "length": 2.1787729263305664, "mode": "train", "epoch": 13}
{"loss": 0.5982010960578918, "acc": 0.6779449582099915, "length": 2.3671875, "mode": "test", "epoch": 13}
{"loss": 0.5863109827041626, "acc": 0.689198911190033, "length": 2.341181993484497, "mode": "train", "epoch": 14}
{"loss": 0.5944950580596924, "acc": 0.6811141967773438, "length": 2.31640625, "mode": "test", "epoch": 14}
{"loss": 0.5756580233573914, "acc": 0.6971224546432495, "length": 2.334861993789673, "mode": "train", "epoch": 15}
{"loss": 0.5853477716445923, "acc": 0.6880522966384888, "length": 2.59765625, "mode": "test", "epoch": 15}
{"loss": 0.5641424059867859, "acc": 0.7090195417404175, "length": 2.4640252590179443, "mode": "train", "epoch": 16}
{"loss": 0.5765358209609985, "acc": 0.6986038088798523, "length": 2.76171875, "mode": "test", "epoch": 16}
{"loss": 0.5509839653968811, "acc": 0.7195172309875488, "length": 2.5760927200317383, "mode": "train", "epoch": 17}
{"loss": 0.5775729417800903, "acc": 0.7030776739120483, "length": 2.6640625, "mode": "test", "epoch": 17}
{"loss": 0.5421064496040344, "acc": 0.7279552817344666, "length": 2.618922233581543, "mode": "train", "epoch": 18}
{"loss": 0.5746711492538452, "acc": 0.7116302847862244, "length": 2.8515625, "mode": "test", "epoch": 18}
{"loss": 0.5338298678398132, "acc": 0.7351696491241455, "length": 2.64845609664917, "mode": "train", "epoch": 19}
{"loss": 0.5639346837997437, "acc": 0.7190735340118408, "length": 2.87109375, "mode": "test", "epoch": 19}
{"loss": 0.52592933177948, "acc": 0.742371678352356, "length": 2.6594011783599854, "mode": "train", "epoch": 20}
{"loss": 0.5620765686035156, "acc": 0.7192748785018921, "length": 2.875, "mode": "test", "epoch": 20}
{"loss": 0.5160428881645203, "acc": 0.7509011626243591, "length": 2.76346755027771, "mode": "train", "epoch": 21}
{"loss": 0.5561591386795044, "acc": 0.7340103387832642, "length": 2.97265625, "mode": "test", "epoch": 21}
{"loss": 0.5079449415206909, "acc": 0.7562291622161865, "length": 2.759932279586792, "mode": "train", "epoch": 22}
{"loss": 0.5502145290374756, "acc": 0.7406630516052246, "length": 3.0234375, "mode": "test", "epoch": 22}
{"loss": 0.5022302865982056, "acc": 0.7601869106292725, "length": 2.8355703353881836, "mode": "train", "epoch": 23}
{"loss": 0.5420799255371094, "acc": 0.7440292239189148, "length": 2.96875, "mode": "test", "epoch": 23}
{"loss": 0.4935046136379242, "acc": 0.7627488970756531, "length": 2.814988613128662, "mode": "train", "epoch": 24}
{"loss": 0.5357165932655334, "acc": 0.7490234375, "length": 3.0703125, "mode": "test", "epoch": 24}
{"loss": 0.4885270595550537, "acc": 0.7651098370552063, "length": 2.8492379188537598, "mode": "train", "epoch": 25}
{"loss": 0.5341213345527649, "acc": 0.7541427612304688, "length": 3.0546875, "mode": "test", "epoch": 25}
{"loss": 0.4841099679470062, "acc": 0.7668083906173706, "length": 2.896578311920166, "mode": "train", "epoch": 26}
{"loss": 0.5345921516418457, "acc": 0.755017101764679, "length": 3.0390625, "mode": "test", "epoch": 26}
{"loss": 0.48105937242507935, "acc": 0.7687503099441528, "length": 2.8790180683135986, "mode": "train", "epoch": 27}
{"loss": 0.5349494218826294, "acc": 0.7511305809020996, "length": 2.9375, "mode": "test", "epoch": 27}
{"loss": 0.47973984479904175, "acc": 0.7670987248420715, "length": 2.8613147735595703, "mode": "train", "epoch": 28}
{"loss": 0.5501719117164612, "acc": 0.7534804344177246, "length": 3.13671875, "mode": "test", "epoch": 28}
{"loss": 0.4798387885093689, "acc": 0.7666944265365601, "length": 2.8679869174957275, "mode": "train", "epoch": 29}
{"loss": 0.5327965021133423, "acc": 0.7553467154502869, "length": 2.9921875, "mode": "test", "epoch": 29}
{"loss": 0.4744615852832794, "acc": 0.7696855068206787, "length": 2.857973098754883, "mode": "train", "epoch": 30}
{"loss": 0.5298566222190857, "acc": 0.755908191204071, "length": 3.0078125, "mode": "test", "epoch": 30}
{"loss": 0.4728928506374359, "acc": 0.7700444459915161, "length": 2.9061925411224365, "mode": "train", "epoch": 31}
{"loss": 0.5295026302337646, "acc": 0.7601348161697388, "length": 2.98828125, "mode": "test", "epoch": 31}
{"loss": 0.4711441695690155, "acc": 0.771757960319519, "length": 2.8953325748443604, "mode": "train", "epoch": 32}
{"loss": 0.5259305238723755, "acc": 0.7621155381202698, "length": 3.01953125, "mode": "test", "epoch": 32}
{"loss": 0.46817928552627563, "acc": 0.7744705677032471, "length": 2.87267804145813, "mode": "train", "epoch": 33}
{"loss": 0.5217236876487732, "acc": 0.7597305774688721, "length": 2.94921875, "mode": "test", "epoch": 33}
{"loss": 0.4656406342983246, "acc": 0.7732049226760864, "length": 2.958099126815796, "mode": "train", "epoch": 34}
{"loss": 0.5297884941101074, "acc": 0.7556961178779602, "length": 2.98828125, "mode": "test", "epoch": 34}
{"loss": 0.4647519290447235, "acc": 0.7739862203598022, "length": 2.8728530406951904, "mode": "train", "epoch": 35}
{"loss": 0.5237909555435181, "acc": 0.7631912231445312, "length": 3.08984375, "mode": "test", "epoch": 35}
{"loss": 0.46139588952064514, "acc": 0.773623526096344, "length": 2.9801113605499268, "mode": "train", "epoch": 36}
{"loss": 0.5220334529876709, "acc": 0.763542115688324, "length": 2.85546875, "mode": "test", "epoch": 36}
{"loss": 0.4614841639995575, "acc": 0.7733361721038818, "length": 2.9226386547088623, "mode": "train", "epoch": 37}
{"loss": 0.5159184336662292, "acc": 0.7643783688545227, "length": 2.96484375, "mode": "test", "epoch": 37}
{"loss": 0.458473265171051, "acc": 0.7763651609420776, "length": 2.9366393089294434, "mode": "train", "epoch": 38}
{"loss": 0.5190951228141785, "acc": 0.761962890625, "length": 2.88671875, "mode": "test", "epoch": 38}
{"loss": 0.45639535784721375, "acc": 0.7778065800666809, "length": 2.9574787616729736, "mode": "train", "epoch": 39}
{"loss": 0.5176510214805603, "acc": 0.7603667974472046, "length": 3.01171875, "mode": "test", "epoch": 39}
{"loss": 0.45394381880760193, "acc": 0.7799609899520874, "length": 3.017744779586792, "mode": "train", "epoch": 40}
{"loss": 0.5161094665527344, "acc": 0.764274537563324, "length": 2.98046875, "mode": "test", "epoch": 40}
{"loss": 0.4513987600803375, "acc": 0.778061032295227, "length": 3.016587495803833, "mode": "train", "epoch": 41}
{"loss": 0.5075078010559082, "acc": 0.7628753185272217, "length": 2.99609375, "mode": "test", "epoch": 41}
{"loss": 0.450313001871109, "acc": 0.7769880294799805, "length": 3.06311297416687, "mode": "train", "epoch": 42}
{"loss": 0.5177267789840698, "acc": 0.7642914056777954, "length": 3.01953125, "mode": "test", "epoch": 42}
{"loss": 0.44583436846733093, "acc": 0.7808136343955994, "length": 3.098126173019409, "mode": "train", "epoch": 43}
{"loss": 0.5145734548568726, "acc": 0.7661819458007812, "length": 3.0390625, "mode": "test", "epoch": 43}
{"loss": 0.44247832894325256, "acc": 0.783901035785675, "length": 3.1676743030548096, "mode": "train", "epoch": 44}
{"loss": 0.5057004690170288, "acc": 0.7680770754814148, "length": 3.19140625, "mode": "test", "epoch": 44}
{"loss": 0.43752503395080566, "acc": 0.7879181504249573, "length": 3.243997573852539, "mode": "train", "epoch": 45}
{"loss": 0.49329549074172974, "acc": 0.7710129022598267, "length": 3.15625, "mode": "test", "epoch": 45}
{"loss": 0.43328985571861267, "acc": 0.7896076440811157, "length": 3.28210711479187, "mode": "train", "epoch": 46}
{"loss": 0.5088481903076172, "acc": 0.7697051763534546, "length": 3.24609375, "mode": "test", "epoch": 46}
{"loss": 0.42794057726860046, "acc": 0.7929564118385315, "length": 3.327012300491333, "mode": "train", "epoch": 47}
{"loss": 0.4950832724571228, "acc": 0.7728256583213806, "length": 3.32421875, "mode": "test", "epoch": 47}
{"loss": 0.4201229512691498, "acc": 0.7981866002082825, "length": 3.410856008529663, "mode": "train", "epoch": 48}
{"loss": 0.486577570438385, "acc": 0.7733047008514404, "length": 3.41015625, "mode": "test", "epoch": 48}
{"loss": 0.41804686188697815, "acc": 0.8002148270606995, "length": 3.4625658988952637, "mode": "train", "epoch": 49}
{"loss": 0.492891788482666, "acc": 0.7752807140350342, "length": 3.515625, "mode": "test", "epoch": 49}
{"loss": 0.4107482433319092, "acc": 0.8046872019767761, "length": 3.43446683883667, "mode": "train", "epoch": 50}
{"loss": 0.49596017599105835, "acc": 0.7765334844589233, "length": 3.53125, "mode": "test", "epoch": 50}
{"loss": 0.4027656614780426, "acc": 0.8092275261878967, "length": 3.4475786685943604, "mode": "train", "epoch": 51}
{"loss": 0.5008186101913452, "acc": 0.7774261236190796, "length": 3.43359375, "mode": "test", "epoch": 51}
{"loss": 0.3977050483226776, "acc": 0.8157089948654175, "length": 3.4350430965423584, "mode": "train", "epoch": 52}
{"loss": 0.4808011054992676, "acc": 0.780505359172821, "length": 3.390625, "mode": "test", "epoch": 52}
{"loss": 0.38665705919265747, "acc": 0.8197800517082214, "length": 3.4477763175964355, "mode": "train", "epoch": 53}
{"loss": 0.47687485814094543, "acc": 0.7817016839981079, "length": 3.421875, "mode": "test", "epoch": 53}
{"loss": 0.38309118151664734, "acc": 0.8223872780799866, "length": 3.4322433471679688, "mode": "train", "epoch": 54}
{"loss": 0.4867287278175354, "acc": 0.7808914184570312, "length": 3.54296875, "mode": "test", "epoch": 54}
{"loss": 0.38012999296188354, "acc": 0.8259273171424866, "length": 3.4572038650512695, "mode": "train", "epoch": 55}
{"loss": 0.4682021141052246, "acc": 0.7871506214141846, "length": 3.5, "mode": "test", "epoch": 55}
{"loss": 0.37585654854774475, "acc": 0.8315066695213318, "length": 3.457501173019409, "mode": "train", "epoch": 56}
{"loss": 0.48912885785102844, "acc": 0.7795897722244263, "length": 3.55078125, "mode": "test", "epoch": 56}
{"loss": 0.3719792366027832, "acc": 0.832706093788147, "length": 3.5020344257354736, "mode": "train", "epoch": 57}
{"loss": 0.4882201850414276, "acc": 0.7867218255996704, "length": 3.578125, "mode": "test", "epoch": 57}
{"loss": 0.3660676181316376, "acc": 0.8339889049530029, "length": 3.532686710357666, "mode": "train", "epoch": 58}
{"loss": 0.4753466844558716, "acc": 0.7931884527206421, "length": 3.51171875, "mode": "test", "epoch": 58}
{"loss": 0.3606758713722229, "acc": 0.8375812768936157, "length": 3.503025770187378, "mode": "train", "epoch": 59}
{"loss": 0.46467065811157227, "acc": 0.7969329357147217, "length": 3.53515625, "mode": "test", "epoch": 59}
{"loss": 0.3497393727302551, "acc": 0.84712153673172, "length": 3.535747528076172, "mode": "train", "epoch": 60}
{"loss": 0.4668370485305786, "acc": 0.8029448986053467, "length": 3.5703125, "mode": "test", "epoch": 60}
test accuracy: 0.7280089855194092
